core/cmd/remote_client.go

```diff
@@ -197,6 +201,73 @@ func (cli *Client) ChangePassword(c *clipkg.Context) (err error) {
 	return nil
 }
 
+// Profile will collect pprof metrics and store them in a folder.
+func (cli *Client) Profile(c *clipkg.Context) error {
+	seconds := c.Uint("seconds")
+	baseDir := c.String("output_dir")
```

Comment:
Perhaps it would be reasonable to use `LOG_FILE_DIR` here and always dump to the same place?

Reply:
I deliberately wanted to detach this from LOG_FILE_DIR as those are logically separate than logs but happy to change